enterprise_subsidy/apps/api/v1/views/transaction.py

```diff
@@ -189,6 +227,12 @@ def base_queryset(self):
             request_based_kwargs.update({"ledger__subsidy__uuid": self.requested_subsidy_uuid})
         if self.requested_transaction_uuid:
             request_based_kwargs.update({"uuid": self.requested_transaction_uuid})
+        if self.requested_subsidy_access_policy_uuid:
+            request_based_kwargs.update({"subsidy_access_policy_uuid": self.requested_subsidy_access_policy_uuid})
+        if self.requested_learner_id:
+            request_based_kwargs.update({"lms_user_id": self.requested_learner_id})
+        if self.requested_content_key:
+            request_based_kwargs.update({"content_key": self.requested_content_key})
```

Comment:
nit
```python
# class-level variable
TRANSACTION_FILTER_KWARGS = {
    'ledger__subsidy__uuid': 'request_subsidy_uuid',
    'uuid': 'requested_transaction_uuid',
    # ...
}
# down in this method...
request_based_kwargs.update({
    filter_kwarg, prop_name
    for filter_kwargs, prop_name in self.TRANSACTION_FILTER_KWARGS.items()
    if getattr(self, prop_name, None)
})
```

Reply:
I think this might be too clever, and also all our other repos do it the less-clever way already